cfgmgr/vlanmgr.cpp

```diff
@@ -324,6 +331,13 @@ void VlanMgr::doVlanTask(Consumer &consumer)
             FieldValueTuple m("mtu", mtu);
             fvVector.push_back(m);
 
+            if(vlan_type != "")
```

Comment:
please add space after `if`, Also can you check for string `empty` as followed in rest of file?